packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/Examples.2/CellContent.6.doc.mdx

```diff
@@ -0,0 +1,24 @@
+import CellContent from './CellContent?dev-site-examples';
+
+# Cell content and cell selection
+
+### Cell content
+
+* The [_content_](/components/cerner-terra-framework-docs/compact-interactive-list/about#cell) of a cell in the [Compact Interactive List](/components/cerner-terra-framework-docs/compact-interactive-list/about) may be empty, text, numerical, or any combination of widgets, but only one focusable element per cell is allowed. 
+  Since Compact Intercative List is one tab stop and arrow keys are used for navigation between cells, these keys are not available to navigate multiple focusable elements inside a cell. 
+* If a cell contains a focusable element inside of it, when navigated to the cell via arrow keys or mouse click, the focusable element will get the focus. 
+* The arrow key press when focused on a cell will navigate away from that cell to the new cell located in corresponding direction.
+
+### Cell selection
+
+This example demonstrates a [Compact Interactive List](/components/cerner-terra-framework-docs/compact-interactive-list/about) that has support for cell selection. 
+
+* A cell is selected by clicking on the cell or pressing the Spacebar on the focused cell. 
+* A cell is selectable unless it contains an interactable element, in that case the interactive element default controls apply.
```

Comment:
Nit:
```suggestion
* A cell is selectable unless it contains an interactable element, in which case the interactive element default controls apply.
```

Reply:
Great suggestion, and much appreciated, but it looks like all wording and styling comment were outdated per this commit by @cldeitrich: https://github.com/cerner/terra-framework/pull/1950/commits/62c72c0098e6edb05ec923b22cf997650ac247bf